docs/source/components/nodes/feature_tracker.rst

```diff
@@ -0,0 +1,152 @@
+FeatureTracker
+==============
+
+FeatureTracker detects key points (features) on a frame and tracks them on the next frame.
+The valid features are obtained from the Harris score or Shi-Tomasi.
+The default number of target features is 320 and the default maximum number of features is 480.
+It supports 720p and 480p resolutions.
+
+
+How to place it
+###############
+
+.. tabs::
+
+  .. code-tab:: py
+
+    pipeline = dai.Pipeline()
+    featureTracker = pipeline.create(dai.node.FeatureTracker)
+
+  .. code-tab:: c++
+
+    dai::Pipeline pipeline;
+    auto featureTracker = pipeline.create<dai::node::FeatureTracker>();
+
+
+Inputs and Outputs
+##################
+
+.. code-block::
+
+               ┌─────────────────┐
+  initialConfig│                 │
+  ────────────►│                 │outputFeatures
+  inputConfig  │     Feature     ├────────────────────►
+  ────────────►│     Tracker     │
+  inputImage   │                 │passthroughInputImage
+  ────────────►│-----------------├────────────────────►
+               └─────────────────┘
+
+
+**Message types**
+
+- :code:`initialConfig` - `FeatureTrackerProperties <https://docs.luxonis.com/projects/api/en/latest/references/python/#depthai.FeatureTrackerProperties>`__
+- :code:`inputConfig` - `FeatureTrackerConfig <https://docs.luxonis.com/projects/api/en/latest/references/python/#depthai.FeatureTrackerConfig>`__
+- :code:`inputImage` - :ref:`ImgFrame`
+- :code:`outputFeatures` - `TrackedFeatures <https://docs.luxonis.com/projects/api/en/latest/references/python/#depthai.TrackedFeatures>`__
+- :code:`passthroughInputImage` - UNKNOWN - TODO
```

Comment:
this is same as inputImage, it's just a passthrough message, so just forwards the message

Reply:
Makes sense, fixed. Are all the other links correct?